src/GeositeFramework/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7-slim
+
+RUN mkdir -p /usr/src/GeositeFramework
```

Comment:
Will this dir already exist by virtue of having been set up as a volume in the docker-compose file?